server.go

```diff
@@ -2559,16 +2559,11 @@ func (s *server) Start() error {
 		// open channels with.
 		if err := s.chanStateDB.PruneLinkNodes(); err != nil {
 			srvrLog.Errorf("Failed to prune link nodes: %v", err)
-
```

Comment:
Are we sure about `PruneLinkNodes`? Wouldn't an error here indicate an issue with the database itself?

Reply:
yeah think you are right - if it fails it's a fatal error, reverted back!